contracts/OptimizedForceMove.sol

```diff
@@ -409,6 +409,101 @@ contract OptimizedForceMove {
 
     }
 
+    function concludeFromOpen(
+        uint256 turnNumRecord,
+        uint256 largestTurnNum,
+        FixedPart memory fixedPart, // don't need appDefinition
+        bytes32 appPartHash,
+        bytes32 outcomeHash,
+        uint8 numStates,
+        uint8[] memory whoSignedWhat,
+        Signature[] memory sigs
+    ) public {
+        // Calculate channelId from fixed part
+        bytes32 channelId = keccak256(
+            abi.encode(fixedPart.chainId, fixedPart.participants, fixedPart.channelNonce)
+        );
+
+        if (turnNumRecord == 0) {
```

Comment:
I noticed that this isn't the same as the equivalent check in `forceMove`, where you did:
```
if (channelStorageHashes[channelId] != bytes32(0)) {
  // ....
```
Was this deliberate?

Reply:
It was not deliberate; in edbd8d3  I have now used the `forceMove` version which is slightly better for gas consumption.